frontend/src/components/course/course.tsx

```diff
@@ -66,12 +67,19 @@ const Course: FC = () => {
   const { courseId, studentId } = useParams();
   const isMentorView = Boolean(studentId);
   const dispatch = useAppDispatch();
+  const navigate = useNavigate();
 
   const isCategoryEditAllowed = checkHasPermission({
     permissionKeys: [PermissionKey.MANAGE_CATEGORIES],
     userPermissions: user?.permissions ?? [],
   });
 
+  const isUserAuthorized = Boolean(user);
+
+  const mentorOrStudentOutputAuthorized =
+    isUserAuthorized && mentorCheckDataStatus === DataStatus.FULFILLED;
+  const mentorOrStudentOutputUnauthorized = !isUserAuthorized;
```

Comment:
Bad naming of variables
1. booleans should start with `is`, `has`
2. I can't understand what are these variables for by looking at the name